soql-analyzer/src/main/scala/com/socrata/soql/AnalysisDeserializer.scala

```diff
@@ -326,7 +326,7 @@ class AnalysisDeserializer[C, T](columnDeserializer: String => C, typeDeserializ
         val seq: NonEmptySeq[SoQLAnalysis[C, T]] = deserializer.read()
         val bt: BinaryTree[SoQLAnalysis[C, T]] = toBinaryTree(seq.seq)
         bt
-      case v if v >= CurrentVersion =>
+      case v if v >= CurrentVersion - 1 =>
```

Comment:
I might not be understanding this right, but based on my understanding this was an accidental with version 7?

maybe we can make a special exception for the mistake version (7), rather than a continuous exception for all versions one below Current?

also I can see how mistakes happen I don't really get this

Reply:
This has changed.  It should be ready for review again.  Everything seems to work although everything is still a mess.